packages/clerk-js/src/core/auth/AuthCookieService.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line import/no-unresolved
```

Comment:
❓ Why we are disabling the eslint rule here, seems that it's not related to the PR

Reply:
@nikosdouvlis  What do you think regarding this? Shall we address it on a different PR?

Reply:
Removed all the unrelated rules.